File: app/src/main/java/online/tatarintsev/currencyrates/model/network/JsonplaceholderApiService.java
package online.tatarintsev.currencyrates.model.network;

import java.util.List;

import io.reactivex.Observable;
import online.tatarintsev.currencyrates.model.data.models.ApiCurrency;
import retrofit2.http.GET;

public interface JsonplaceholderApiService {

    @GET("users")
    Observable<List<ApiCurrency>> getCurrencies();

}

File: app/src/main/java/online/tatarintsev/currencyrates/model/network/JsonplaceholderApiService.kt
package online.tatarintsev.currencyrates.model.network

import io.reactivex.Observable
import online.tatarintsev.currencyrates.model.data.models.ApiCurrency
import retrofit2.http.GET

interface JsonplaceholderApiService {
    @GET("users")
    fun getCurrencies(): Observable<List<ApiCurrency>>
}